src/CandidateList.js

```diff
@@ -0,0 +1,139 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+import CandidateCard from './CandidateCard';
+import { useCookies } from 'react-cookie';
+import LoginPage from './LoginPage';
+import SignUpPage from './SignUpPage';
+const dataURL = 'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:8080/candidates';
+function CandidateList() {
+    const [candidates, setCandidates] = useState(null);
+    const [isLogIn, setIsLogIn] = useState(false);
+    const [isSignUpFinished, setIsSignUpFinished] = useState(false);
+    const [isLogInPage, setIsLogInPage] = useState(false);
+    const [isSignUpPage, setIsSignUpPage] = useState(false);
+    const [cookies, removeCookie] = useCookies(['token']);
+    async function getCandidates() {
+        const { data } = await axios.get(dataURL);
+        setCandidates(data);
+    }
+    const handleLogInOutButtonClick = () => {
+        if(isLogIn) {
+            removeCookie('token');
+            setIsLogIn(false);
+            setIsLogInPage(false);
+            setIsSignUpPage(false);
+        }
+        else {
+            setIsLogInPage(true);
+            setIsSignUpPage(false);
+        }
+    }
+    const handleShowResultButtonClick = () => {
+        setIsLogInPage(false);
+        setIsSignUpPage(false);
+    }
+    const handleSignUpButtonClick = () => {
+        setIsLogInPage(false);
+        setIsSignUpPage(true);
+    }
+    useEffect(() => {
+        getCandidates();
+        cookies.token === 'undefined' ? setIsLogIn(false) : setIsLogIn(true);
```

Comment:
Truthy/Falsy 를 응용해봐도 좋겠죠?

```suggestion
        !cookies.token ? setIsLogIn(false) : setIsLogIn(true);
```

혹은 가독성은 좀 떨어집니다만
```suggestion
        setIsLogIn(!!cookies.token);
```

요렇게 해봐도 좋겠군요

Reply:
훨씬 좋네요 !!! 밑에 껀 처음 보는 것 같아요 ! undefined를 true/false로 바꿔줄 수도 있군요 !!!